tasks/buildah.yaml

```diff
@@ -79,12 +79,38 @@ spec:
     volumeMounts:
     - mountPath: /var/lib/containers
       name: varlibcontainers
+  - image: registry.fedoraproject.org/fedora:35
```

Comment:
I tried using `registry.access.redhat.com/ubi9/python-39` as a base image for this, but since it uses the `default` user, I was getting a permission error on L105. Not sure if there's a better image to use to run this Python script.

Reply:
is that a typo @brunoapimentel or did you really use `ubi9` instead of `ubi8` ?

That aside, yeah, the official python image shipped with the openshift samples is `registry.redhat.io/ubi8/python-39:latest` 

Next, by `default` user do you mean the `default` service account?  If so, then yeah, I would think the pipeline runs that use this task will need to use the `pipeline` service account in order to have sufficient permissions.

Otherwise, I would need to look at pipeline run / pod yaml, events, and pod logs.

But let's see if @Michkov @ejegrova have anything to add.

Reply:
Same issue happens on ubi8, btw.

Reply:
```
      securityContext:
        runAsUser: 0
```
Should fix it.